frontend/app/src/store/session/state.ts

```diff
@@ -28,7 +29,8 @@ export const defaultState: () => SessionState = () => ({
   lastBalanceSave: 0,
   lastDataUpload: 0,
   timeframe: TimeFramePeriod.ALL,
-  showUpdatePopup: false
+  showUpdatePopup: false,
+  animationsEnabled: isAnimationsEnabled()
```

Comment:
If you see `isAnimationEnabled` function, I stored the value on localStorage. Previously I tried to use `useLocalStorage` but I need it to reactive between components (App.vue and FrontendSettings.vue), it cannot be used for this purpose (so I put it at store).

I guess I'll just use `useLocalStorage` at `Login.vue` and leave this part.

Reply:
> Would it be possible to split it to two different options? Keep the button in the login screen, make it properly persist on restarts and have it only affect the login screen itself but not progress.
> 
> And then the other option in the settings can only affect the progress animations instead.

Updated

Reply:
> If you see `isAnimationEnabled` function, I stored the value on localStorage. Previously I tried to use `useLocalStorage` but I need it to reactive between components (App.vue and FrontendSettings.vue), it cannot be used for this purpose (so I put it at store).
> 
> I guess I'll just use `useLocalStorage` at `Login.vue` and leave this part.

Since these are two different options you can now use them for both. What was the exact problem?

Reply:
I think you misunderstood. Just ignore the animation at login screen since it is handled on its own page.

The problem is between `App.vue` (where i put the css to reduce the animation, I put it here cause it is the most top component), and with `FrontendSettings.vue` (where I put the setting to control the state). Using `useLocalStorage` won't make the state between both component synced.

Reply:
> I think you misunderstood. Just ignore the animation at login screen since it is handled on its own page.
> 
> The problem is between `App.vue` (where i put the css to reduce the animation, I put it here cause it is the most top component), and with `FrontendSettings.vue` (where I put the setting to control the state). Using `useLocalStorage` won't make the state between both component synced.

So if you have an instance of `useLocalStorage('setting')` in `App.vue` and one in the `FrontendSettings.vue` those two instances won't sync between them? is that the problem?

Reply:
Yes that's what I mean.